tests/integration/components/mirage-handler-test.js

```diff
@@ -76,7 +76,7 @@ module('Integration | Component | mirage-handler', function (hooks) {
     }),
   };
   for (let [type, file] of Object.entries(SCENARIOS)) {
-    test(`upload handler throws if invalid ${type} is provided`, async function (assert) {
+    skip(`upload handler throws if invalid ${type} is provided`, async function (assert) {
```

Comment:
Wondering if we should only skip the test for `type === 'audio'`. The other two scenarios were stable, weren't they? Something like:

```js
     (type === 'audio' ? skip : test)(`upload handler throws if invalid ${type} is provided`, async function (assert) {
```

Reply:
Good point. Will add this now

Reply:
I had to skip both `audio` and `video` types as they both fail in the same way

Reply:
Oh. I missed that. Having both failing in the same way actually makes it less confusing. 😄